react/components/LogoutCustomerSession.js

```diff
@@ -14,50 +14,75 @@ import TelemarketingIcon from '../icons/TelemarketingIcon'
 
 /** Component that shows the client info calls the setSession function  to logout. */
 export default class LogoutCustomerSession extends Component {
+  
   handleHeaderRendering = () => {
-    const { client } = this.props
+    const { client, mobile } = this.props
 
     return (
-      <div className="flex align-center">
-        <CustomerIcon />
-        <div className="pa2">
-          {client.email ? `${truncateString(client.email, 25)}` : null}
+      mobile ? (
+        <div className="flex align-center w-100">
```

Comment:
To avoid structure duplication, the conditional could be used only inside de className and inside the  vtex-telemarketing__client-name-bar to decide what to render (name or email)